standalone/build.gradle

```diff
@@ -63,3 +64,9 @@ configurations {
 application {
     mainClassName = 'com.tngtech.keycloakmock.standalone.Main'
 }
+
+buildConfig {
+    buildConfigField('String', 'NAME', "\"${project.name}\"")
+    buildConfigField('String', 'VERSION', "\"${project.version}\"")
+    buildConfigField('long', 'BUILD_TIME', "${System.currentTimeMillis()}L")
```

Comment:
Hm ... might be nice to have, but it's currently unused. What do you think?

Reply:
you mean only the build time or all three lines?

Reply:
Only the build time. The other two are used in the main class of the standalone server.

Reply:
well ... we can add it when we find a place to use it ;-)

Reply:
So you mean to say:
```suggestion
```